docs/source/conf.py

```diff
@@ -26,7 +26,7 @@
 copyright = '2020, labscript suite'
 author = 'labscript suite contributors'
 
-version = __version__
+version = __version__.split('+')[0]
```

Comment:
What was the motivation behind this? Isn't it good to include the commit hash in the version of the latest build?

Reply:
It was so that the local preview would mimic what appears online, although for Read the Docs this will only ever be latest, stable. or a release version. I guess I was still thinking of github-pages hosted docs, where the dev number appears every time, without the commit hash, e.g. https://labscript-suite.github.io. Happy to remove this.